examples/apiproduct/dogs-cats/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: user-secret-apikey
+  annotations:
+    secret.kuadrant.io/user-id: user-01
+  labels:
+    secret.kuadrant.io/managed-by: authorino
```

Comment:
Authorino may cache this API key secret when it reconciles the `AuthConfig` because Kuadrant controller will [set label selectors](https://github.com/Kuadrant/kuadrant-controller/blob/356fb4d7abce66ef2ad5d93bad6461ee6c254e02/pkg/authproviders/authorino/provider.go#L82) that match the ones here. This part is OK.

However, this is NOT enough for Authorino to watch changes related to this API key secret, independently from the reconciliation of the `AuthConfig` (e.g. key rotation, label update on the secret, deletion of the secret). To put such events within the scope of the Authorino secret reconciler, matching [`SecretLabelSelectors`](https://github.com/Kuadrant/authorino-operator/blob/febc0828aaee302096bf770db535fc85941e5fc8/api/v1beta1/authorino_types.go#L119) must as well be specified in the spec of the `Authorino` CR that is [created by kuadrantctl](https://github.com/Kuadrant/kuadrantctl/blob/21fb6c74ea58534c7561098fb473f90d085f128e/pkg/authorino/authorino.go#L19).

If not specified, Authorino will default to watching only secret-related events of resources that include labels matching `authorino.kuadrant.io/managed-by=authorino`.

Reply:
thanks for the tip. I will update to use the default value then.

Reply:
added changes to https://github.com/Kuadrant/kuadrantctl/pull/35/commits/470741435286181b5b880c5fbe0716423991461d